File: C/Vetores/Numeros_pares.c
#include <stdio.h>
#include <string.h>


int main ()
{

    int N, i, pares;

    printf ("Quantos numeros voce vai digitar? ");
    scanf ("%d", &N);

    int vet[N];

    for (i = 0; i < N; i++)
    {
        printf ("Digite um numero: ");
        scanf ("%d", &vet[i]);
    }


    printf ("\nNUMEROS PARES:\n");

    pares = 0;

    for (i = 0; i < N; i++)
    {
        if (vet[i] % 2 == 0)
        {
            printf ("%d  ", vet[i]);
            pares = pares + 1;
        }
    }
    printf ("\n");
    printf ("\nQUANTIDADE DE PARES = %d\n", pares);




    return 0;
}

File: Python/Vetores/Numeros_pares.py
N = int(input("Quantos numeros voce vai digitar? "))

vet = [0 for x in range(N)]

for i in range(0, N):
    vet[i] = int(input("Digite um numero: "))


print()
print("NUMEROS PARES:")

cont = 0

for i in range(0, N):
    if vet[i] % 2 == 0:
        cont = cont + 1
        print(vet[i], end= "  ")

print()
print(f"\nQuantidade de pares = {cont}")


